### PR TITLE
Menu mappings

### PR DESCRIPTION
**Pull request by NeunEinser**
~~Has overlapping on the MouseSettingsScreen with #610~~, this is more focused on menu screens, while #610 is more focused on options.

### CODE REVIEW

File: mappings/net/minecraft/client/options/GameOption.mapping [hunk 1]
```diff
@@ -3,6 +3,7 @@ CLASS cve net/minecraft/client/options/GameOption
 	FIELD B CHAT_COLOR Lcup;
 	FIELD C CHAT_LINKS Lcup;
 	FIELD D CHAT_LINKS_PROMPT Lcup;
+	FIELD E DISCRETE_MOUSE_SCROLL Lcup;
```
Comment (YanisBft) [inline mappings/net/minecraft/client/options/GameOption.mapping]: Overlap #610, and the ingame name is "discrete scrolling"

Comment (NeunEinser) [reply mappings/net/minecraft/client/options/GameOption.mapping]: Ah, I missed that

File: mappings/net/minecraft/client/options/ExponentialGameOption.mapping [hunk 2]
```diff
@@ -0,0 +1 @@
+CLASS cva net/minecraft/client/options/ExponentialGameOption
```
Comment (asiekierka) [inline mappings/net/minecraft/client/options/ExponentialGameOption.mapping]: ???

Comment (NeunEinser) [reply mappings/net/minecraft/client/options/ExponentialGameOption.mapping]: It's used by the new scroll sensitivity slider in the mouse settings. While all other sliders in option menus are linear, this slider uses an exponential/logarithmic scale. Roughly the first three quarters are 0 to 1, then it goes up to 10.

No idea how else to call this.

Comment (Runemoro) [reply mappings/net/minecraft/client/options/ExponentialGameOption.mapping]: I think it's called a "[logarithmic scale](https://en.wikipedia.org/wiki/Logarithmic_scale)", I've never heard of "exponential scale". So I think `LogarithmicGameOption` would be better.

Comment (NeunEinser) [reply mappings/net/minecraft/client/options/ExponentialGameOption.mapping]: It also uses Math.log(), so I changed it now.

**Pull request merged**